src/site/articles/doc-comment-guidelines/index.markdown

```diff
@@ -36,6 +36,12 @@ See also these related articles:
 
 ## Examples
 
+Dart supports two syntaxes for doc comments (`///` and `/**`), but we
```

Comment:
I'd rather not explanation the reasons for preferring /// over /** here—the style guide seems like the place for that. Maybe we could do something like this:

```
Here's a typical comment for a function or method:
...
<aside class="alert alert-info" markdown="1">
<b>Style note:</b>
Dart supports two syntaxes for doc comments: `///` and `/**`.
This document follows the
[style guide recommendation](/articles/style-guide/#do-use-doc-comments-when-commenting-members-and-types)
of using `///`. Older source code tends to use `/**`.
</aside>
```

Then we could add the justification to the style guide. (You could do that in this pull request, or I can do it separately.)